src/python/library/CMakeLists.txt

```diff
@@ -147,12 +178,13 @@ if(${TRITON_ENABLE_PYTHON_GRPC})
   )
 endif() # TRITON_ENABLE_PYTHON_GRPC
 
-install(
-  CODE "file(GLOB _Wheel \"${CMAKE_CURRENT_BINARY_DIR}/generic/triton*.whl\")"
-  CODE "file(INSTALL \${_Wheel} DESTINATION \"${CMAKE_INSTALL_PREFIX}/python\")"
-)
 
-if (NOT WIN32)
+if(WIN32)
+  install(
+    CODE "file(GLOB _Wheel \"${CMAKE_CURRENT_BINARY_DIR}/windows/triton*.whl\")"
```

Comment:
Add an additional variable WHEEL_DIR, and set it to 
"${CMAKE_CURRENT_BINARY_DIR}/windows/" or "${CMAKE_CURRENT_BINARY_DIR}/linux/" in the ifdef block. 
And pull the 
CODE "file(GLOB _Wheel \"${WHEEL_DIR}/triton*.whl\")" 
block outside the ifdef. The good practice is to keep the ifdef as short as possible.